HerPublicWebsite.UnitTests/BusinessLogic/QuestionnaireUpdaterTests.cs

```diff
@@ -207,13 +208,15 @@ public async Task GenerateReferralAsync_WhenCalled_UpdatesQuestionnaireReferralD
     }
     
     [Test]
-    public async Task GenerateReferralAsync_WhenCalledWithEmail_SendOneEmailWithReferralCode()
+    public async Task GenerateReferralAsync_WhenCalledWithEmailAndLocalAuthorityIsLive_SendOneEmailWithReferralCodeWithLiveTemplate()
     {
         // Arrange
         const string testCustodianCode = "1234";
-        const string testReferralCode = "referral code";
+        const int testReferralId = 12;
         const string testName = "Example Person";
         const string testEmailAddress = "test@example.com";
+        new Mock<Questionnaire>().Setup(p => p.LocalAuthorityHug2Status).Returns(LocalAuthorityData.Hug2Status.Live);
```

Comment:
You're not actually using this `Mock` that you create, since you don't assign it to a variable.